gtfs-realtime/spec/en/reference.md

```diff
@@ -1,14 +1,29 @@
 A GTFS Realtime feed lets transit agencies provide consumers with realtime information about disruptions to their service (stations closed, lines not operating, important delays, etc.) location of their vehicles, and expected arrival times.
 
-Version 1.0 of the feed specification is discussed and documented on this site.
+Version 2.0 of the feed specification is discussed and documented on this site.
 
 ### Term Definitions
 
-*   **required**: Exactly one
-*   **repeated**: Zero or more
```

Comment:
I'm concerned that `repeated` has been removed; I think it should be included so that people know they can supply 0 or more.

Reply:
I have identity theft issue that's why I can't share next any thing without my secret code (CHAUDHARY) copy that 
![Screenshot_20241102-181346](https://github.com/user-attachments/assets/ae24a872-5841-4f1f-bd10-200dca5e99a9)